tests/common.py

```diff
@@ -105,6 +110,16 @@ def test_checks_disable_one_by_one_with_cli(self):
             real_errors = self.get_count_code_errors(all_check_errors)
             assertDictEqual(self, real_errors, expected_errors)
 
+    def test_checks_disable_one_by_one_with_env(self):
+        for check2disable in self.expected_errors:
+            expected_errors = self.expected_errors.copy()
+            sys.argv = ["", "--no-exit", "--no-verbose"] + self.file_paths
+            os.environ[DISABLE_ENV_VAR] = check2disable
```

Comment:
I think the os.environ are not cleaned between unittests

I think you will need to use with method to reset them or mock or something else


FYI I have created this way with odoo-pre-commit-vauxoo tool